fabfile.py

```diff
@@ -1092,6 +1091,21 @@ def app_compile_nodbupdate():
     execute(compile_javascript)
 
 
+@task
+def setup_nginx_file():
+    """Creates nginx config file from template."""
+    sudo("cp %s/doc/sample_nginx_config/assembl.yourdomain.com /etc/nginx/sites-available/assembl.%s" % (env._projectpath, env.public_hostname))
+    sudo("ln -s /etc/nginx/sites-available/assembl.%s /etc/nginx/sites-enabled/" % env.public_hostname)
+    sudo("/etc/init.d/nginx restart")
+
+
+@task
+def setup_assembl_service():
+    sudo("cp %s/doc/sample_systemd_script/assembl.service /etc/systemd/system/assembl.service" % env.public_hostname)
```

Comment:
You should make the service file a jinja2 template, as the project path, the assembl user, etc are all variables based on the RC files. You should render the template and put the template with the right permissions in the systemd directory.